lib/rubocop/cop/performance/ancestors_include.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Performance
+      # This cop is used to identify usages of `ancestors.include?` and
+      # change them to use `<=` instead.
+      #
+      # @example
+      #   # bad
+      #   A.ancestors.include?(B)
+      #
+      #   # good
+      #   A <= B
+      #
+      class AncestorsInclude < Cop
+        include RangeHelp
+
+        MSG = 'Use `<=` instead of `ancestors.include?`.'
+
+        def_node_matcher :ancestors_include_candidate?, <<~PATTERN
+          (send (send $_child :ancestors) :include? $_parent)
```

Comment:
This is a nitpick feedback. Can you update the names?

```suggestion
          (send (send $_subclass :ancestors) :include? $_superclass)
```

In Ruby there is a method named `superclass`, so it may be a bit better as a term. E.g.: `Class.superclass #=> Module`